ARM-wvd-templates/Create and provision WVD host pool/README.md

```diff
@@ -0,0 +1,112 @@
+Click the button below to deploy:
+
+<a href="https://portal.azure.com/#create/Microsoft.Template/uri/https%3A%2F%2Fraw.githubusercontent.com%2FAzure%2FRDS-Templates%2Fmaster%2FARM-wvd-templates%2FCreate%20and%20provision%20WVD%20host%20pool%2FmainTemplate.json" target="_blank">
+    <img src="http://azuredeploy.net/deploybutton.png"/>
+</a>
+<a href="http://armviz.io/#/?load=https%3A%2F%2Fraw.githubusercontent.com%2FAzure%2FRDS-Templates%2Fmaster%2FARM-wvd-templates%2FCreate%20and%20provision%20WVD%20host%20pool%2FmainTemplate.json" target="_blank">
+    <img src="http://armviz.io/visualizebutton.png"/>
+</a>
+
+# ARM Template to Create and provision new Windows Virtual Desktop hostpool
+

```

Comment:
I think it would be helpful to have a note saying this template is for for non-Classic WVD only and link to the v1 template.
FYI I added a note "This ARM template only supports Windows Virtual Desktop (classic), which doesn't support Azure Resource Manager Windows Virtual Desktop objects." to the [v1 readme here](https://github.com/Azure/RDS-Templates/blob/staging/wvd-templates/Create%20and%20provision%20WVD%20host%20pool/README.md)

Reply:
ok added